front/src/modules/companies/hooks/useUpdateCompanyBoardColumns.ts

```diff
@@ -100,7 +101,7 @@ export function useUpdateCompanyBoard() {
           orderedPipelineStages?.map((pipelineStage) => ({
             id: pipelineStage.id,
             title: pipelineStage.name,
-            colorCode: pipelineStage.color,
+            colorCode: pipelineStage.color as ThemeColor,
```

Comment:
I'm a bit annoyed by all these `as` casting. Is there a way to replace that by the right string literal in graphql? or to perform run-time guards?
We have avoided putting as much `as` as possible in the codebase so far, I feel this is opening the deamons gate!

Reply:
GraphQL codegen generates TS enum on top of GraphQL enums, the best approach is to use a typeguard yes !